### PR TITLE
[16.0][MIG] partner_aging

### PR DESCRIPTION
**Pull request by cormaza**

### CODE REVIEW

**Comment by yboujraf** [comment]
Dear @cormaza 
Do you know why the module is not released ?
BR

**Comment by cormaza** [comment]
> Dear @cormaza Do you know why the module is not released ? BR

It should be approved at least by two reviewers, then some maintainer can merge

**Comment by miquelalzanillas** [comment]
Hi @cormaza ,

Could you please rebase?

Thanks!

**Review by miquelalzanillas — APPROVED** [review]
Code and functional review. LGTM.

**Comment by miquelalzanillas** [comment]
> Dear @cormaza Do you know why the module is not released ? BR

Hi @yboujraf ,

You can tests this module on runboat using the link here.

image

If your tests are ok, you can approve this pull request using this.

image

image

Thanks!

**Review by ppyczko — APPROVED** [review]
Please, rename the pull request and commit following this pattern: [16.0][MIG] \<module\>: Migration to 16.0

Otherwise, LGTM, tested in runboat

**Review by miquelpascual — APPROVED** [review]
LGTM, tested in runboat

**Comment by miquelalzanillas** [comment]
Hi @pedrobaeza ,

Could you review please?

File: partner_aging/readme/USAGE.rst
Comment (peluko00) [inline partner_aging/readme/USAGE.rst:5]: Can you fix the itinerary to access to the module:
```
#. Go to Invoicing
#. Click on Customers > Customer Aging or Vendors > Supplier Aging
#. Change the date if necessary
```

**Review by peluko00 — CHANGES_REQUESTED** [review]

**Comment by miquelalzanillas** [comment]
Hi @pedrobaeza ,

we need to start migration to v17, do you think this is ready to merge?

Thanks,

**Comment by pedrobaeza** [comment]
/ocabot migration partner_aging

I don't use this module. A PSC should review it.

**Comment by miquelalzanillas** [comment]
Hi @HaraldPanten ,

Could you check if we can merge this module?

Thanks,

**Comment by HaraldPanten** [comment]
> Hi @HaraldPanten ,
> 
> Could you check if we can merge this module?
> 
> Thanks,

Hi @miquelalzanillas I'm not using this module neither, but I see that there's something wrong with the commit history. You should check it 👍

**Comment by miquelalzanillas** [comment]
> > Hi @HaraldPanten ,
> > Could you check if we can merge this module?
> > Thanks,
> 
> Hi @miquelalzanillas I'm not using this module neither, but I see that there's something wrong with the commit history. You should check it 👍

Hi @cormaza , can you check it?

Thanks.

**Comment by cormaza** [comment]
> > Hi @HaraldPanten ,
> > Could you check if we can merge this module?
> > Thanks,
> 
> Hi @miquelalzanillas I'm not using this module neither, but I see that there's something wrong with the commit history. You should check it 👍

Hi @HaraldPanten, thank you for review, I can't see commit history problem, can you specific where is it?

**Comment by HaraldPanten** [comment]
I see... It seems that it's due to the name change to partner_aging.

Even though, you're not following the proper name for the migration commit. Could you check it?

**Comment by miquelalzanillas** [comment]
Hi @cormaza ,

I think what Harald means is to update your migration commit message from `[16.0][MIG] partner_aging` to `[MIG][16.0] partner_aging:Migration to 16.0`. 

Can you do it please?

Thanks!

**Comment by HaraldPanten** [comment]
> Hi @cormaza ,
> 
> I think what Harald means is to update your migration commit message from `[16.0][MIG] partner_aging` to `[MIG][16.0] partner_aging:Migration to 16.0`.
> 
> Can you do it please?
> 
> Thanks!

That's it. THX Miquel.

**Comment by HaraldPanten** [comment]
Oops, the name should appear without the module version, sorry. It should have this name:

`[MIG] partner_aging: Migration to 16.0`

**Comment by HaraldPanten** [comment]
Merging, as it has several validations

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
@miquelalzanillas done 👍

**Comment by miquelalzanillas** [comment]
Thanks @HaraldPanten & @cormaza !

**Pull request merged**
